pkg/oci/pack.go

```
@@ -0,0 +1,306 @@
+		header.Name = filepath.ToSlash(rel)
+
+		if err := tw.WriteHeader(header); err != nil {
+			return fmt.Errorf("writing tar header for %s: %w", rel, err)
+		}
+
+		if !info.IsDir() {
+			f, err := os.Open(path)
+			if err != nil {
+				return fmt.Errorf("opening %s: %w", rel, err)
+			}
+			defer f.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `f.Close` is not checked

(errcheck)